react/index.tsx

```diff
@@ -0,0 +1,7 @@
+import DeviceDetector from './DeviceDetector'
+import useDevice from './useDevice'
+
+export default {
```

Comment:
I think you don't need this, since the other exports are using `export default` and babel will transform:

```diff
- import { useDevice } from 'vtex.device-detector'
+ import useDevice from 'vtex.device-detector/useDevice'
```

Reply:
didn't work

Reply:
it did work eventually by resetting the workspace